### DISCUSSION TITLE
Group update notifications for the same mod

### DISCUSSION THREAD

**Discussion by CodeF53** [modrinth/.github#55] [Feature requests]
**Is your feature request related to a problem? Please describe.**
When a mod updates more than once, I dont really care about the older updates; I just want to see what mods have updated in general.
Even after trying to dismiss duplicate notifications for the same mods, I didn't notice that fabric api was still here twice.
![image](https://user-images.githubusercontent.com/37855219/129949955-4e95615d-fd67-435e-82d2-1e793f105c58.png)

**Describe the solution you'd like**
Ideally, I would only have one notification for every mod that updates regardless of how many times they have updated
Instead of displaying like the above screenshot, mods would be ordered by most recent update, with a number saying how many times they have updated, and a button to view all the updates I have missed.
![image](https://user-images.githubusercontent.com/37855219/129951048-b8484c13-0f74-4f52-b7c5-60cd38993a69.png)

**Comment by CodeF53** [comment]
updated mockup for staging branch
![image](https://user-images.githubusercontent.com/37855219/155257298-7e1de0da-c649-4524-a09b-1b520b7cae71.png)

**Comment by CodeF53** [comment]
Since this requires a fair bit of both frontend and backend work, I will leave this issue open here for now

**Comment by Andre601** [comment]
One thing that should be kept in mind (And probably implemented) is a way to view all the update notifications, as a mod may post multiple updates - one for each MC-version - which may hide the one you're actually interested in.

And while you're add it should notifs be auto-dismissed as soon as you click them. It's annoying when you click them, get the new update (or don't) but the notification is still there, requiring you to manually dismiss it.

**Comment by intergrav** [comment]
This was a concept I made for it. I didn't realize this issue existed lol

![image](https://user-images.githubusercontent.com/42325132/189505424-5aa78ac7-4a0f-48d9-9b69-cd56eaed594f.png)

**Comment by falseresync** [comment]
Instead of nested cards it should just be a list of dates to versions, no need to duplicate information

**Comment by falseresync** [comment]
Like so, wouldn't even need collapsing
![Frame 7](https://user-images.githubusercontent.com/17195973/189525572-50336ce2-5c6f-41b7-9cb0-2e70ef5d8118.png)

**Comment by falseresync** [comment]
*bump to the top* I still want this \:)

**Comment by triphora** [comment]
Implemented as of modrinth/knossos#1232, will be on production soon(tm)
![](https://cdn.discordapp.com/attachments/734077874708938867/1132779690566877234/image.png)
